README.md

```diff
@@ -15,19 +15,20 @@ is a fork of Spree.
 Solidus actually consists of several different gems, each of which are maintained
 in a single repository and documented in a single set of
 [online documentation](http://docs.solidus.io/). By requiring the
-solidus gem you automatically require all of the necessary gem dependencies which are:
+[`solidus`](https://rubygems.org/gems/solidus) gem you automatically require all
+of the necessary gem dependencies which are:
 
-* solidus\_api (RESTful API)
-* solidus\_frontend (Cart and storefront)
-* solidus\_backend (Admin area)
-* solidus\_core (Essential models, mailers, and classes)
-* solidus\_sample (Sample data)
+* [`solidus_api`](https://rubygems.org/gems/solidus_api) (RESTful API)
```

Comment:
I'm not sure the rubygems page provides useful information. Maybe we should use github links with the intention of adding READMEs to the subprojects